aspnet/mvc/views/forms/sample/final/Views/DemoReg/Register.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@model FormsTH.ViewModels.RegisterViewModel
+@addTagHelper "*, Microsoft.AspNet.Mvc.TagHelpers"
+
+<form asp-controller="DemoReg" asp-action="Register" method="post">
+    <div asp-validation-summary="ValidationSummary.All"></div>
```

Comment:
Use `ValidationSummary.ModelOnly`. Bit silly to have property errors show up twice.